LogMonitor/src/LogMonitor/Main.cpp

```diff
@@ -6,7 +6,57 @@
 #include "pch.h"
 #include "Version.h"
 
-using namespace std;
+#include "opentelemetry/sdk/version/version.h"
+#include "opentelemetry/trace/provider.h"
+
+#include "opentelemetry/exporters/ostream/span_exporter_factory.h"
+#include "opentelemetry/sdk/trace/simple_processor_factory.h"
+#include "opentelemetry/sdk/trace/tracer_provider_factory.h"

```

Comment:
can we have all this in pch?

Reply:
Separated the code for handling telemetry and getting the system info into different files